lib/zuora/objects/payment_method.rb

```diff
@@ -2,7 +2,7 @@ module Zuora::Objects
   class PaymentMethod < Base
     belongs_to :account
 
-    # validates_presence_of :account_id
+    # validates_presence_of :account_id, :unless => Proc.new { |contact| contact.new_record? }
```

Comment:
Do we want to keep this commented out? It is uncommented upstream: https://github.com/zuorasc/zuora/blob/master/lib/zuora/objects/payment_method.rb#L5

Reply:
This was commented as part of the work we had done internally as seen [here](https://github.com/Sage/zuora/pull/2).